learning_service/Taskfile.yml

```diff
@@ -0,0 +1,64 @@
+version: '3'
+
+vars:
+  BUILDER_IMAGE: learning_service_image
+
+tasks:
+  prepare:
+    - task: venv_create
+    - task: venv_activate
+    - task: pip_install
+  venv_create:
+    cmds:
+      - python3 -m venv venv
+  venv_activate:
```

Comment:
Does the virtual environment, activated by the Task task, stay activated after Task finishes? The `source` command executes the script and loads stuff from it without creating a subshell, which allows us to use stuff from it. However, launching Task will create its own subshell for the process, which means that the stuff, loaded by `source` will be lost. Or is it only for the purpose of being used by the `prepare` task?

Reply:
The purpose is just to set up the environment. I will add a `deactivate` so that the shell will exit.